### PR TITLE
Fix msgTypes + Add sort_by/sort_order args to MessageFilter/PostFilter

### PR DESCRIPTION
**Pull request by philogicae**
# Found issues:
- Missing sort_by/sort_order filters: [issue 95 on SDK](https://github.com/aleph-im/aleph-sdk-python/issues/95)
- message_types passed as arg to MessageFilter breaks if nb elements > 1

# Proposed solution:
- Modified MessageFilter/PostFilter + Added SortBy/SortOrder enums
- Fix typo on msgType -> msgTypes
- Also requires merging of [PR](https://github.com/aleph-im/pyaleph/pull/555) on PyAleph repo

@hoh

### CODE REVIEW

**Review by MHHukiewitz — DISMISSED** [review]
.Thank you for the contribution! Also, good finds on msgTypes and enum_as_str :+1:

**Comment by philogicae** [comment]
Fixed the isort test failing

File: src/aleph/sdk/utils.py [hunk 1]
```diff
@@ -115,12 +115,14 @@ async def copy_async_readable_to_buffer(
...
+def enum_as_str(obj: Optional[Union[str, Enum]]) -> str:
     """Returns the value of an Enum, or the string itself when passing a string.
 
     Python 3.11 adds a new formatting of string enums.
     `str(MyEnum.value)` becomes `MyEnum.value` instead of `value`.
     """
+    if obj is None:
+        return ""
     if not isinstance(obj, str):
         raise TypeError(f"Unsupported enum type: {type(obj)}")
```
Comment (MHHukiewitz) [inline src/aleph/sdk/utils.py:127]: As it stands, just returning an empty string seems risky to me. Please revert the change, if accomodating for the Optional[str] in the return type is too much effort to be worth it.

**Review by MHHukiewitz — CHANGES_REQUESTED** [review]

File: src/aleph/sdk/utils.py (hunk 1 above)
Comment (philogicae) [reply src/aleph/sdk/utils.py]: You are right, I reverted the change. Makes more sense to keep the Optional returned value. But I had to modify chains/common, the tests require null safety.

**Review by MHHukiewitz — APPROVED** [review]

**Pull request merged**